staging/src/k8s.io/smd/README.md

```diff
@@ -0,0 +1,5 @@
+# Structured Merge and Diff
```

Comment:
Can we change the name of the package? `mergediff`? Something that is more easily to guess if you've never seen the full name first.